packages/react-i18n/src/utilities/translate.tsx

```diff
@@ -45,10 +45,12 @@ export interface TranslateOptions<Replacements = {}> {
 }
 
 function numberFormatCacheKey(
-  locale: string,
-  options: Intl.NumberFormatOptions = {},
+  locales?: string | string[],
+  options?: Intl.NumberFormatOptions,
 ) {
-  return `${locale}${JSON.stringify(options)}`;
+  return `${
+    Array.isArray(locales) ? locales.join('-') : locales
+  }-${JSON.stringify(options)}`;
```

Comment:
`JSON.stringify(undefined)` returns `undefined` then adding undefined to the interpolation will do something like.

`fr-undefined` is it the behaviour we want?

Reply:
it's probably fine. i'm fine changing to `options ?? ''` if we want to use empty string instead